polyfills/Object/isSealed/config.toml

```diff
@@ -10,7 +10,7 @@ edge = "<13"
 edge_mob = "<13"
 firefox = "<35"
 firefox_mob = "<35"
-ie = "*"
+ie = "<9"
```

Comment:
These methods chsnged in es6 and ie does not support that version of these methods